README.md

```diff
@@ -165,7 +165,7 @@ Apprentices should memorize [**this**](Cheatsheet.md) within the first month.
 1. Coding assessment
 1. App development assessment
 1. [Recursion](/recursion/recursion.md)
-1. [Asset Optimization](/optimization/optimization.md)
+1. [Asset Optimization](/electives/optimization.md)
```

Comment:
Should this be moved to the elective section if it is an elective now, @vegetabill ?

Reply:
Hi @FrozenGirl-spd , just move this to the electives section and you'll be ready for merge.